### DISCUSSION TITLE
Set Static Values for Plot Key

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#1279] [Ideas]
In a lot of data-display use cases there may be a possible minimum or maximum value for counts the dataset that never changes regardless of what specific data we are looking at. For example, in confusion matrix's each square in this polygon has a minimum possible value of 0 and a maximum possible value of the sum of the 4 squares. However, our slider for the `color` uses the minimum and maximum values of the current data.
![image](https://user-images.githubusercontent.com/4793561/156435726-879b3d16-cc39-4761-834b-70f07035e483.png)

This could be `color_value_min` and `color_value_max` values?

**Comment by mturoci** [comment]
Tracked in https://github.com/h2oai/wave/issues/1280
